docs/net/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Ethereum for .NET Developers
+meta:
+  - name: description
+    content: Learn how to develop for Ethereum using rust-based projects and tooling
+  - property: og:title
+    content: Ethereum for .NET Developers
+  - property: og:description
+    content: Learn how to develop for Ethereum using .NET-based projects and tooling
+lang: en-US
+sidebar: auto
+sidebarDepth: 0
+---
+
+# Ethereum for .NET Developers
+
+<div class="featured">Learn how to develop for Ethereum using .NET-based projects and tooling</div><br>
+
+Use Ethereum to create decentralized applications (or "dapps") that utilize the benefits of cryptocurrency and blockchain technology. These dapps can be trustworthy, meaning that once they are deployed to Ethereum, they will always run as programmed. They can control digital assets in order to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them and are nearly impossible to censor.
+
+<img src="https://i.imgur.com/onFLM4L.png" width="100%" />
```

Comment:
Did you create this image yourself? It displays massive on desktop & the text is hard to read:

![Image 2019-11-05 at 3 43 35 PM](https://user-images.githubusercontent.com/8097623/68255792-1b2b6a80-ffe3-11e9-966b-68a38a54bd07.png)

Let's remove this unless you can provide a cleaner image of similar dimensions to /python & /java (~240px height):
![Image 2019-11-05 at 3 45 00 PM](https://user-images.githubusercontent.com/8097623/68255863-4615be80-ffe3-11e9-882f-5c5cb82fc3e5.png)